pilot/pkg/features/security.go

```diff
@@ -88,6 +88,11 @@ var (
 		return res
 	}()
 
+	EnableMultiClusterNodeAuthorizer = env.Register(
+		"ENABLE_MULTICLUSTER_NODE_AUTHORIZER",
+		false,
+		"If enabled, node authorization will be performed for CSRs originating from remote clusters.").Get()
```

Comment:
would this be useful for external cp as well? i recall that is another scenario.

If so, should we call it external_node_authorizer?

Reply:
Yes! This can work with external cp as well. I think it could be renamed to ENABLE_EXTERNAL_NODE_AUTHORIZER? working on it.

Reply:
Thanks!